spark/src/main/scala/org/apache/spark/sql/delta/catalog/DeltaTableV2.scala

```diff
@@ -189,6 +192,23 @@ case class DeltaTableV2(
       deltaLog, info.options, spark.sessionState.conf.useNullsForMissingDefaultColumnValues)
   }
 
+  /**
+   * Starts a transaction for this table, using the snapshot captured during table resolution.
+   *
+   * WARNING: Caller is responsible to ensure that table resolution was recent (e.g. if working with
+   * [[DataFrame]] or [[DeltaTable]] API, where the table could have been resolved long ago).
+   */
+  def startTransactionWithInitialSnapshot(): OptimisticTransaction =
+    startTransaction(Some(snapshot))
+
+  /**
+   * Starts a transaction for this table, using Some provided snapshot, or a fresh snapshot if None
```

Comment:
s/Some/some

Reply:
That was intentional... Some/None as in `Option`.